homeassistant/components/launch_library/config_flow.py

```diff
@@ -0,0 +1,27 @@
+"""Config flow to configure launch library component."""
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class LaunchLibraryFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for Launch Library component."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
```

Comment:
We're missing a type annotation for the user input. We try to type the whole signature when adding type annotations. We can use `dict[str, Any] | None`.